ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceRpcApisTest.java

```diff
@@ -387,6 +392,8 @@ public JsonRpcHttpService getJsonRpcHttpService(final boolean[] enabledNetServic
     WebSocketConfiguration webSocketConfiguration = WebSocketConfiguration.createDefault();
     P2PNetwork p2pNetwork = mock(P2PNetwork.class);
     MetricsConfiguration metricsConfiguration = MetricsConfiguration.builder().build();
+    NatService natService = mock(NatService.class);
+    lenient().when(natService.isNatEnvironment()).thenReturn(false);
```

Comment:
If my understanding is correct, `lenient` is only necessary if mockito detects that the stub isn't used. Is it complaining about `isNatEnvironment` not being called?

Reply:
I just removed it